docs/documentation/aurora-config.md

```diff
@@ -214,11 +214,19 @@ The following baseImage are in use at NTA
+| path                 | default | description                                                                                                                                                                                                                                                                     |
+| -------------------- | ------- | ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| template             |         | Name of template in default namespace to use. This is required if type is template                                                                                                                                                                                              |
+| templateFile         |         | Set the location of a local template file. It should be in the templates subfolder. This is required if type is localTemplate                                                                                                                                                   |
+| `parameters/<KEY>`   |         | The parameters option is used to set values for parameters in the template. If the template has either of the parameters VERSION, NAME, SPLUNK_INDEX or REPLICAS, the values of these parameters will be set from the standard version, name and replicas AuroraConfig options. |
+| replicas             |         | If set will override replicas in template                                                                                                                                                                                                                                       |
+| resources/cpu/min    |         | Specify minimum/request cpu. 1000m is 1 core.                                                                                                                                                                                                                                   |
+| resources/cpu/max    |         | Specify maximum/limit cpu.                                                                                                                                                                                                                                                      |
+| resources/memory/min |         | Specify minimum/request memory.                                                                                                                                                                                                                                                 |
```

Comment:
Burde vi legge til informasjon om hvilke verdier som kan settes? 

- Memory: 100Mi, 2Gi
- CPU: 2, 100m

Kan kanskje legge til disse linkene også?
https://kubernetes.io/docs/concepts/configuration/manage-compute-resources-container/#meaning-of-cpu
https://kubernetes.io/docs/concepts/configuration/manage-compute-resources-container/#meaning-of-memory